Fixtures/Miscellaneous/SwiftPMXCTestHelper/Tests/SwiftPMXCTestHelper/SwiftPMXCTestHelperTests.swift

```diff
@@ -0,0 +1,17 @@
+import XCTest
+@testable import SwiftPMXCTestHelper
+
+class SwiftPMXCTestHelperTests: XCTestCase {
+
+    func testExample1() {
+    }
+
+    func test_Example2() {
+    }
+
+    func testExample3(arg: String) {
+    }
+
+    func nontest() {
+    }
+}
```

Comment:
It would be a good idea to have an Obj-C test as well, since the XCTest framework can behave slightly differently there.